presto-main/src/main/java/com/facebook/presto/operator/window/WindowAnnotationsParser.java

```diff
@@ -37,12 +38,15 @@ private WindowAnnotationsParser() {}
     {
         WindowFunctionSignature[] signatures = clazz.getAnnotationsByType(WindowFunctionSignature.class);
         checkArgument(signatures.length > 0, "Class is not annotated with @WindowFunctionSignature: %s", clazz.getName());
+        WindowFunctionOptions[] options = clazz.getAnnotationsByType(WindowFunctionOptions.class);
```

Comment:
This should `getAnnotations()` since there can be at most one instance of the annotation (it's not repeatable)